### 3. Verifique se o nível de uso pode ser considerado o mesmo para firmas TOP e Não TOP


In [1]:
# Importar bibliotecas necessárias
import pandas as pd
from scipy import stats

In [2]:
# Local do arquivo
caminho = '/Users/fabicampanari/Desktop/_8-Prova Matematematica/1-statiscalMeasures_ Hypothesis Testing II./1🇧🇷-statiscalMeasures_ Hypothesis Testing II./MATEMA/Biscobis.xlsx'

# Carregar o arquivo para o DataFrame
df = pd.read_excel(caminho)

In [3]:
# Visualizar as primeiras linhas do DataFrame para entender a estrutura dos dados
print("Primeiras linhas do DataFrame:")
print(df.head())

Primeiras linhas do DataFrame:
   ident   x1   x2   x3   x4   x5   x6   x7  x8  x9  x10  x11  x12  x13  x14
0      1  4.1  0.6  6.9  4.7  2.4  2.3  5.2   0  32  4.2    1    0    1    1
1      5  6.0  0.9  9.6  7.8  3.4  4.6  4.5   0  58  6.8    1    0    1    3
2      7  4.6  2.4  9.5  6.6  3.5  4.5  7.6   0  46  5.8    1    0    1    1
3      9  5.5  1.6  9.4  4.7  3.5  3.0  7.6   0  63  5.4    1    0    1    3
4     11  2.4  1.6  8.8  4.8  2.0  2.8  5.8   0  32  4.3    1    0    0    1


In [4]:
# Verificar as colunas disponíveis no DataFrame
print("Colunas disponíveis no DataFrame:")
print(df.columns)

Colunas disponíveis no DataFrame:
Index(['ident', 'x1', 'x2', 'x3', 'x4', 'x5', 'x6', 'x7', 'x8', 'x9', 'x10',
       'x11', 'x12', 'x13', 'x14'],
      dtype='object')


In [5]:
# Supondo que a coluna 'x13' indica se a firma é TOP ou Não TOP
# Se as colunas corretas forem diferentes, substitua 'x13' pelo nome correto da coluna

# Verificar os valores únicos na coluna 'x13' para identificar as firmas TOP e Não TOP
print("Valores únicos na coluna 'x13':")
print(df['x13'].unique())

Valores únicos na coluna 'x13':
[1 0]


In [6]:
# Supondo que 1 representa firmas TOP e 0 representa firmas Não TOP
firmas_top = df[df['x13'] == 1]
firmas_nao_top = df[df['x13'] == 0]


In [7]:
# Supondo que a coluna 'x14' representa o nível de uso
# Se a coluna correta for diferente, substitua 'x14' pelo nome correto da coluna

# Selecionar os níveis de uso para firmas TOP e Não TOP
uso_top = firmas_top['x14']
uso_nao_top = firmas_nao_top['x14']

In [8]:
# Passo 7: Realizar o Teste t de Student para Amostras Independentes
# Realizar o teste t de Student para amostras independentes
t_stat, p_val = stats.ttest_ind(uso_top, uso_nao_top, equal_var=False)
print("\nResultados do Teste t de Student:")
print(f"Estatística t: {t_stat}")
print(f"Valor p: {p_val}")



Resultados do Teste t de Student:
Estatística t: 0.480761973820412
Valor p: 0.6317572451539955


In [9]:
# Passo 8: Interpretação e Conclusão
# Interpretação e Conclusão
if p_val < 0.05:
    print("\nConclusão: Rejeitamos a hipótese nula. O nível de uso é diferente para firmas TOP e Não TOP.")
else:
    print("\nConclusão: Não rejeitamos a hipótese nula. O nível de uso é o mesmo para firmas TOP e Não TOP.")


Conclusão: Não rejeitamos a hipótese nula. O nível de uso é o mesmo para firmas TOP e Não TOP.


Resultados do Teste t de Student:
### Estatística t (t_stat): 0.480761973820412
### Valor p (p_val): 0.6317572451539955

**Interpretação:**
O valor p obtido é 0.6317572451539955, que é maior que o nível de significância comum de 0.05. Isso significa que não há evidências estatísticas suficientes para rejeitar a hipótese nula.

**Conclusão:**
Não há evidências estatísticas suficientes para concluir que o nível de uso é significativamente diferente entre firmas TOP e Não TOP. Portanto, não rejeitamos a hipótese nula. O nível de uso é o mesmo para firmas TOP e Não TOP.
